test/e2e/specs/editor/various/block-bindings.spec.js

```diff
@@ -1248,50 +1248,125 @@ test.describe( 'Block bindings', () => {
 					page.locator( '#paragraph-binding' )
 				).toBeHidden();
 			} );
+
+			test( 'should add empty paragraph block when pressing enter', async ( {
+				editor,
+				page,
+			} ) => {
+				await editor.insertBlock( {
+					name: 'core/paragraph',
+					attributes: {
+						content: 'paragraph default content',
+						metadata: {
+							bindings: {
+								content: {
+									source: 'core/post-meta',
+									args: { key: 'text_custom_field' },
+								},
+							},
+						},
+					},
+				} );
+				const paragraphBlock = editor.canvas.getByRole( 'document', {
+					name: 'Block: Paragraph',
+				} );
+				await paragraphBlock.click();
+				await page.keyboard.press( 'Enter' );
+				const initialParagraph = ( await editor.getBlocks() )[ 0 ];
```

Comment:
Nit: you could write: 

```js
const [ initialParagraph, newEmptyParagraph ] = await editor.getBlocks()
```

Reply:
Thank you! Changed in [this commit](https://github.com/WordPress/gutenberg/pull/59361/commits/cdc48dad62b9783d119c4ad41367686f6c330b55).